stripe.go

```diff
@@ -485,3 +480,59 @@ func initUserAgent() {
 	}
 	encodedStripeUserAgent = string(marshaled)
 }
+
+// Bool returns a pointer to the bool value passed in.
+func Bool(v bool) *bool {
+	return &v
+}
+
+// BoolValue returns the value of the bool pointer passed in or
+// false if the pointer is nil.
+func BoolValue(v *bool) bool {
+	if v != nil {
+		return *v
+	}
+	return false
+}
+
+// String returns a pointer to the string value passed in.
+func String(v string) *string {
```

Comment:
Would you mind if we just alphabetized these new functions relative to each other for nicety?